src/utils/runlog.py

```diff
@@ -202,22 +239,22 @@ def _write_runlog(self):
             # write the runlog to a csv file
 
             file_name = csv_filenames["main"]
-            file_path = f"{main_path}/{file_name}"
-            df = read_hdfs_csv(file_path)
+            file_path = str(os.path.join(self.main_path, file_name))
+            df = self.read_func(file_path)
             newdf = df.append(self.runlog_main_df)
-            write_hdfs_csv(file_path, newdf)
+            self.write_func(file_path, newdf)
 
             file_name = csv_filenames["configs"]
-            file_path = f"{main_path}/{file_name}"
-            df = read_hdfs_csv(file_path)
+            file_path = str(os.path.join(self.main_path, file_name))
```

Comment:
The logs seem to be skipping a line for every row. This might be hard to index or convert to hive later. 
![image](https://github.com/ONSdigital/research-and-development/assets/121172811/06fd32ee-4875-4805-a72c-d2e032f5d2ef)

Reply:
Yes, this is the case in my csv files as well, and I also have this happening when I write the full_responses dataframe to csv, which is strange!